backend/src/main/java/com/woowacourse/friendogly/club/service/ClubCommandService.java

```diff
@@ -1,11 +1,87 @@
 package com.woowacourse.friendogly.club.service;
 
 
+import com.woowacourse.friendogly.club.domain.Club;
+import com.woowacourse.friendogly.club.dto.request.SaveClubMemberRequest;
+import com.woowacourse.friendogly.club.dto.request.SaveClubRequest;
+import com.woowacourse.friendogly.club.dto.response.SaveClubMemberResponse;
+import com.woowacourse.friendogly.club.dto.response.SaveClubResponse;
+import com.woowacourse.friendogly.club.repository.ClubMemberRepository;
+import com.woowacourse.friendogly.club.repository.ClubRepository;
+import com.woowacourse.friendogly.exception.FriendoglyException;
+import com.woowacourse.friendogly.member.domain.Member;
+import com.woowacourse.friendogly.member.repository.MemberRepository;
+import com.woowacourse.friendogly.pet.domain.Pet;
+import com.woowacourse.friendogly.pet.repository.PetRepository;
+import java.util.List;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Transactional
 public class ClubCommandService {
 
+    private final ClubRepository clubRepository;
+    private final MemberRepository memberRepository;
+    private final PetRepository petRepository;
+
+    public ClubCommandService(
+            ClubRepository clubRepository,
+            MemberRepository memberRepository,
+            PetRepository petRepository,
+            ClubMemberRepository clubMemberRepository) {
+        this.clubRepository = clubRepository;
+        this.memberRepository = memberRepository;
+        this.petRepository = petRepository;
+    }
+
+    public SaveClubResponse save(Long memberId, SaveClubRequest request) {
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new FriendoglyException("회원 정보를 찾지 못했습니다."));
+
+        Club newClub = clubRepository.save(request.toEntity(member));
+
+        List<Pet> participatingPets = request.participatingPetsId().stream()
+                .map(id -> petRepository.findById(id).orElseThrow(() -> new FriendoglyException("강아지 정보를 찾지 못했습니다.")))
+                .toList();
+
+        List<String> petImageUrls = participatingPets.stream()
+                .map(Pet::getImageUrl)
+                .toList();
+
+        newClub.addClubPet(participatingPets);
+
+        return new SaveClubResponse(newClub, 1, petImageUrls);
+    }
+
+    public SaveClubMemberResponse saveClubMember(Long clubId, Long memberId, SaveClubMemberRequest request) {
+        Club club = clubRepository.findById(clubId)
+                .orElseThrow(() -> new FriendoglyException("모임 정보를 찾지 못했습니다."));
```

Comment:
```suggestion
    public SaveClubMemberResponse addClubMember(Long clubId, Long memberId, SaveClubMemberRequest request) {
        Club club = clubRepository.findById(clubId)
                .orElseThrow(() -> new FriendoglyException("모임 정보를 찾지 못했습니다."));
```

`ClubMember`라는 매핑 테이블을 만들기는 했지만 `ClubCommandService` 입장에서 이 메서드를 보면 결국에는 Member를 Club에 추가하는 역할을 하고 있네요. `add`라는 네이밍은 어떨까요?

Reply:
여기에 덧붙여서, 현재 `saveClubMember()` 메서드는 모임에 참여할 회원 뿐만 아니라, 모임에 참여할 강아지 정보도 추가하고 있습니다.
현재 네이밍이 Pet이 추가된다는 의미를 충분히 나타내지 못하는 것 같아요.
Club에 Pet이 추가되는 동작이 가려지지 않도록 네이밍을 고려해보시면 좋겠습니다.

Reply:
비즈니스적인 내용이 메서드에 드러나도 괜찮다면, participate라는 네이밍이 간결하고 적절한 것 같은데 어떠신가요?